src/Management/ResourceBuilder.php

```diff
@@ -410,6 +412,14 @@ private function buildUser(array $data): User
         ]);
     }
 
+    private function buildOrganization(array $data): Organization
+    {
+        return $this->createObject(Organization::class, [
+            'name' => $data['name'],
+            'sys' => $this->buildSystemProperties($data['sys'])
```

Comment:
We're digressing a bit, but PSR-2 has some grey areas, and I usually prefer `@Symfony` simply because it extends PSR-2 and covers some of these grey areas 🙂